src/style.css

```diff
@@ -1,3 +1,52 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-    background-color: bisque;
-}
\ No newline at end of file
+  background-color: bisque;
+}
+
+.logo {
+  width: 60px;
+  height: 60px;
+}
+
+.logo-wrap {
+  display: flex;
+}
+
+.selection-wrap {
+  display: flex;
+  column-gap: 20px;
+  width: 300px;
+  margin-top: 15px;
+}
+
+.header-main-wrap {
+  display: flex;
+  justify-content: space-between;
+  background-color: rgb(118, 110, 110);
+}
```

Comment:
- [ ] You can add some padding to the `header-main-wrap` so the contents can come out better

This is what it currently looks like 
![image](https://user-images.githubusercontent.com/55185309/162785883-57b7520e-083f-462b-9c06-e652fa59bcd1.png)

what it looks like with padding 
![image](https://user-images.githubusercontent.com/55185309/162786313-392c846e-7e33-4318-bd4c-390a8ce734b6.png)